src/ui/locales/ms.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Togol atribusi",
+    "AttributionControl.MapFeedback": "Maklum balas peta",
+    "FullscreenControl.Enter": "Masukkan skrin penuh",
+    "FullscreenControl.Exit": "Keluar dari skrin penuh",
+    "GeolocateControl.FindMyLocation": "Cari lokasi saya",
+    "GeolocateControl.LocationNotAvailable": "Lokasi tidak tersedia",
+    "LogoControl.Title": "Logo peti peta",
+    "Map.Title": "Peta",
+    "NavigationControl.ResetBearing": "Tetapkan semula galas ke utara",
+    "NavigationControl.ZoomIn": "Mengezum masuk",
+    "NavigationControl.ZoomOut": "Zum keluar",
+    "ScrollZoomBlocker.CtrlMessage": "Gunakan ctrl + tatal untuk mengezum peta",
```

Comment:
change to `"Gunakan ⌘ + tatal untuk mengezum peta"`

`"⌘"` or `"ctrl"` modifier depends on OS (?)